plugins/base/src/main/kotlin/parsers/factories/DocTagsFromIElementFactory.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("IMPLICIT_CAST_TO_ANY")
```

Comment:
Can you please move it to specific methods? It may be misleading if somebody writes a new part